src/Authentication/Actions/CreateIdentityTrait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace CodeIgniter\Shield\Authentication\Actions;
+
+use CodeIgniter\Shield\Entities\User;
+use CodeIgniter\Shield\Models\DatabaseException;
+use CodeIgniter\Shield\Models\UserIdentityModel;
+
+/**
+ * Create an identity for auth action
+ */
+trait CreateIdentityTrait
```

Comment:
Why create a new trait? The User entity already has several methods dealing with traits. Seems to me like it should just go there.

Reply:
Because I thought that the two methods in actions are the same knowledge.

Since these methods are only used by Actions, there is no need for `User` to have them.

Reply:
With the slight refactoring I mentioned earlier the method could be useful in more situations than the current one to all developers.

Reply:
I will remove this Trait in the next PR.